doc/src/advanced.rst

```diff
@@ -22,15 +22,15 @@ However if an error is raised when selecting the folder, the connection may be
 left open.
 
 IMAPClient may be used as a context manager that automatically closes
-connections when they are not needed anymore::
+connections when they are not needed any more::
 
   import imapclient
 
   with imapclient.IMAPClient(host="imap.foo.org") as c:
       c.login("bar@foo.org", "passwd")
       c.select_folder("INBOX")
 
-Watching a mailbox using idle
+Watching a Mailbox Using IDLE
```

Comment:
Does the capital M is consistent with other mentions of "mailbox" everywhere else?

Reply:
It's capitalised because this is a heading. Otherwise, I agree that just "mailbox" is correct.